schema.graphql

```diff
@@ -29,6 +29,10 @@ type Tranche @entity {
   index: BigInt!
   # The amount of collateral in this tranche for redemption
   totalCollateral: BigInt!
+  # The amount of collateral in this tranche for redemption at time bond matured
+  totalCollateralAtMaturity: BigInt!
```

Comment:
Should these be optional (no exclamation point) given they dont exist until maturity?

Reply:
Ah that might make more sense. I figured they'd be initialised at 0 until they were set